src/ContaoManager/Plugin.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Terminal42\AutoRegistrationBundle\ContaoManager;
+
+use Contao\CoreBundle\ContaoCoreBundle;
+use Contao\ManagerPlugin\Bundle\BundlePluginInterface;
+use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
+use Contao\ManagerPlugin\Bundle\Config\ConfigInterface;
+use Contao\ManagerPlugin\Bundle\Parser\ParserInterface;
+use Terminal42\AutoRegistrationBundle\Terminal42AutoRegistrationBundle;
+
+
+class Plugin implements BundlePluginInterface
+{
+
+    /**
+     * Gets a list of autoload configurations for this bundle.
+     *
+     * @param ParserInterface $parser
+     *
+     * @return ConfigInterface[]
+     */
+    public function getBundles(ParserInterface $parser)
+    {
+        return [
+            BundleConfig::create(Terminal42AutoRegistrationBundle::class)
```

Comment:
I guess we should use a `->setReplace(['autoregistration'])` here? This will correctly solve module dependencies if a Contao 3 bundle requires `autoregistration` in the `autoload.ini`.